### PR TITLE
CSS Brochure Form

### PR DESCRIPTION
**Pull request by louiewhitz**
Screen Shot 2022-08-24 at 1 37 59 PM

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.container {
+  width: vw20;
```
Comment (BrettAlbright) [inline css-brochure-form/styles.css]: This does not appear to be a valid width setting, you would want `20vw` indicating 20% of the viewport width, although I would avoid ever using it in any real production code. Typically containers are set sizes like pixels

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  margin: 30%;
```
Comment (BrettAlbright) [inline css-brochure-form/styles.css]: if you want to use margin to center a container on the screen, you would want to go with `margin 0 auto` that would apply the available space outside the width of the container as margin evenly to both sides which will center the container within the available space.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Very nice work on this one, the image is very very close to the example. you only want a couple of small updates to the CSS to give it a more solid foundation.

File: css-brochure-form/styles.css (hunk 2 above)
Comment (louiewhitz) [reply css-brochure-form/styles.css]: You're not wrong, coming right up!

**Comment by louiewhitz** [comment]
Screen Shot 2022-08-24 at 2 52 58 PM

**Review by BrettAlbright — APPROVED** [review]
👍🏼  Great work!

**Pull request merged**